ui/components/app/edit-gas-fee-popover/network-statistics/status-slider/status-slider.js

```diff
@@ -14,57 +15,70 @@ const GRADIENT_COLORS = [
   '#9A4D71',
   '#B44561',
   '#C54055',
+  '#D73A49',
```

Comment:
The gradient was missing a final color which meant if `networkCongestion` rounded to 100% the label and arrow would be black.